src/badge/badge.js

```diff
@@ -1,20 +1,31 @@
 import React from 'react';
-import { Text, View, StyleSheet } from 'react-native';
+import { Text, View, StyleSheet, TouchableOpacity } from 'react-native';
 
 let styles = {};
```

Comment:
I'm not sure I understand this. I would still need to import `TouchableOpacity` no?